src/ui-shell/sidenav/sidenav-item.component.ts

```diff
@@ -69,12 +70,24 @@ export class SideNavItem {
 	 */
 	@Output() navigation = new EventEmitter<Promise<boolean>>();
 
+	/**
+	 * Emits when `active` input is changed. This is mainly used to indicate to any parent menu items that a
+	 * child sidenav item is active or not active.
+	 */
+	@Output() toggled = new EventEmitter<boolean>();
+
```

Comment:
```suggestion
	/**
	 * Emits when `active` input is changed. This is mainly used to indicate to any parent menu items that a
	 * child sidenav item is active or not active.
	 */
	@Output() selected = new EventEmitter<boolean>();

```

This lines up better with our existing conventions, and what's actually going on here

Reply:
Also multi-line change suggestions! This is the best 😁